modules/farm_fd2_school/src/entrypoints/Vue1/App.vue

```diff
@@ -136,7 +111,33 @@
   </div>
 </template>
 
-<script></script>
+<script>
+export default {
+  data() {
+    return {
+      selectedCrop: { id: 4, label: 'RADISH' },
```

Comment:
Setting this gets easier if you use an array of strings here instead of an array of objects.  You'll see that in the next assignment.